share/picongpu/examples/FoilLCT/include/picongpu/simulation_defines/param/components.param

```diff
@@ -50,7 +54,7 @@ namespace picongpu
      *
      * Adjust the settings of the selected profile in laser.param
```

Comment:
please add the new laser profile to the comment above in *all* examples and the default `components.param`

Reply:
yes, some of the `laser.param` files in examples are off-style. Pls try to fit into each as it looks in its surroundings, we can clean them up in a follow-up PR where they are inconsistent.